catalog/region-restriction-templates/DenyRegions.yaml

```diff
@@ -37,6 +40,9 @@
     - "s3:GetAccountPublic*"
     - "s3:ListAllMyBuckets"
     - "s3:PutAccountPublic*"
+    %{ if servicequotas_enabled == "true" }
+    - "servicequotas:*"
+    %{ endif }
```

Comment:
You need to handle `servicequotas_enabled` just like `cloudfront_lambda_edge_enabled`